front/components/sparkle/navigation.tsx

```diff
@@ -84,7 +86,7 @@ export const topNavigation = ({
     nav.push({
       id: "settings",
       label: "Admin",
-      icon: KeyIcon,
+      icon: Cog6ToothIcon,
```

Comment:
oups 😬

Reply:
I think we have consensus :)

Reply:
Yes, but sorry for updating the header and not the side menu !